### PR TITLE
Update Lesson_6_Give_Your_Token_Identity.md

### PR DESCRIPTION
**Pull request by shungo0222**
I fixed some typos and code.

`await findMetadataPda(mint)` -> `metaplex.nfts().pdas().metadata({mint})`
Because `findMetadataPda` is deprecated.

### CODE REVIEW

File: Solana_Core/en/Core_2/Lesson_6_Give_Your_Token_Identity.md [hunk 1]
```diff
@@ -15,7 +15,6 @@ import {
   keypairIdentity,
   bundlrStorage,
   toMetaplexFile,
-  findMetadataPda,
 } from "@metaplex-foundation/js"
```
Comment (AlmostEfficient) [inline Solana_Core/en/Core_2/Lesson_6_Give_Your_Token_Identity.md:18]: Why remove `findMetadataPda`?

File: Solana_Core/en/Core_2/Lesson_6_Give_Your_Token_Identity.md [hunk 2]
```diff
@@ -71,7 +70,7 @@ async function createTokenMetadata(
   console.log("metadata uri:", uri)
 
   // get metadata account address
-  const metadataPDA = await findMetadataPda(mint)
+  const metadataPDA = metaplex.nfts().pdas().metadata({mint})
```
Comment (AlmostEfficient) [inline Solana_Core/en/Core_2/Lesson_6_Give_Your_Token_Identity.md:73]: I'm confused

**Review by AlmostEfficient — CHANGES_REQUESTED** [review]

File: Solana_Core/en/Core_2/Lesson_6_Give_Your_Token_Identity.md (hunk 2 above)
Comment (shungo0222) [reply Solana_Core/en/Core_2/Lesson_6_Give_Your_Token_Identity.md:73]: I replied below.

**Review by AlmostEfficient — APPROVED** [review]

**Comment by AlmostEfficient** [comment]
Ooof Metaplex team going at the speed of light again

**Pull request merged**